deps/rabbit/src/rabbit_msg_store.erl

```diff
@@ -859,19 +1163,22 @@ handle_call({new_client_state, CRef, CPid, MsgOnDiskFun, CloseFDsFun}, _From,
 handle_call({client_terminate, CRef}, _From, State) ->
     reply(ok, clear_client(CRef, State));
 
-handle_call({read, MsgId}, From, State) ->
-    State1 = read_message(MsgId, From, State),
-    noreply(State1);
+%% @todo DONENOW
+%handle_call({read, MsgId}, From, State) ->
+%    State1 = read_message(MsgId, From, State),
+%    noreply(State1);
```

Comment:
while this should not be used anymore, the client code may still do a call on read_1 so it's probably best to leave it as before for now